coral/src/app/features/configuration/clusters/AddNewClusterForm.test.tsx

```diff
@@ -0,0 +1,291 @@
+import { cleanup, screen, within } from "@testing-library/react";
+import { userEvent } from "@testing-library/user-event";
+import AddNewClusterForm from "src/app/features/configuration/clusters/AddNewClusterForm";
+import { customRender } from "src/services/test-utils/render-with-wrappers";
+import { addNewCluster } from "src/domain/cluster";
+
+jest.mock("src/domain/cluster/cluster-api.ts");
+
+const mockAddNewClusterRequest = addNewCluster as jest.MockedFunction<
+  typeof addNewCluster
+>;
+
+const mockedUsedNavigate = jest.fn();
+jest.mock("react-router-dom", () => ({
+  ...jest.requireActual("react-router-dom"),
+  useNavigate: () => mockedUsedNavigate,
+}));
+
+const mockedUseToast = jest.fn();
+jest.mock("@aivenio/aquarium", () => ({
+  ...jest.requireActual("@aivenio/aquarium"),
+  useToast: () => mockedUseToast,
+}));
+
+describe("<AddNewClusterForm />", () => {
+  const originalConsoleError = console.error;
+
+  beforeEach(() => {
+    console.error = jest.fn();
```

Comment:
This will swallow all `console.errors` which can hide actual errors or mistakes. `console.error` should only be mocked right in the scope where we also assert that it's called and that's it.

Reply:
[2d81c7b](https://github.com/Aiven-Open/klaw/pull/2301/commits/2d81c7b1d0b82d0097e78cd76f8b3a7ef0475b54)

Reply:
🙏